pkg/api/common/versions.go

```diff
@@ -11,71 +11,71 @@ import (
 // AllKubernetesSupportedVersions is a whitelist map of all supported Kubernetes version strings
 // The bool value indicates if creating new clusters with this version is allowed
 var AllKubernetesSupportedVersions = map[string]bool{
-	"1.6.6":          true,
-	"1.6.9":          true,
-	"1.6.11":         true,
-	"1.6.12":         true,
-	"1.6.13":         true,
-	"1.7.0":          true,
-	"1.7.1":          true,
-	"1.7.2":          true,
-	"1.7.4":          true,
-	"1.7.5":          true,
-	"1.7.7":          true,
-	"1.7.9":          true,
-	"1.7.10":         true,
-	"1.7.12":         true,
-	"1.7.13":         true,
-	"1.7.14":         true,
-	"1.7.15":         true,
+	"1.6.6":          false,
+	"1.6.9":          false,
+	"1.6.11":         false,
+	"1.6.12":         false,
+	"1.6.13":         false,
+	"1.7.0":          false,
+	"1.7.1":          false,
+	"1.7.2":          false,
+	"1.7.4":          false,
+	"1.7.5":          false,
+	"1.7.7":          false,
+	"1.7.9":          false,
+	"1.7.10":         false,
+	"1.7.12":         false,
+	"1.7.13":         false,
+	"1.7.14":         false,
+	"1.7.15":         false,
 	"1.7.16":         true,
```

Comment:
Should we keep 1.7.16?

Reply:
Do we know if any acs-engine users still provision 1.7 clusters? Otherwise, I'd say get rid of it except that 1.7.x is still supported in AKS currently. (I assume we can't remove Kubernetes versions that AKS still expects.)